components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -595,6 +595,11 @@ IJsonSerializer jsonSerializer
                                 }
                             }
 
+                            //  refresh the Tag if a new one is provided
+                            if (!string.IsNullOrEmpty(request.Tag)) {
```

Comment:
With `string.IsNullOrEmpty()` we provide no way of resetting or removing a tag. If we only check for the string being `null` then the client will have a way to set it to an empty string. Is this something we should think about ?